src/person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(id, name = 'Unknown', age = 18, parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Please kindly implement a method to add rentals as stated in the requirement, kindly check this [link for better understanding](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). 🙏 

<img width="497" alt="g126" src="https://github.com/alexansaa/School-Library/assets/103386183/eb91a585-6e85-48b1-8970-93ecf713a6f5">